hudi-common/src/main/java/org/apache/hudi/metadata/HoodieMetadataMergedLogRecordReader.java

```diff
@@ -53,38 +52,16 @@
 
   private static final Logger LOG = LogManager.getLogger(HoodieMetadataMergedLogRecordReader.class);
 
-  // Set of all record keys that are to be read in memory
-  private Set<String> mergeKeyFilter;
```

Comment:
@prashantwason : we are removing mergeKeyFilter as we don't see any usage of it. wanted to confirm that its ok to remove it ?